plugins/modules/notification/cisco_webex.py

```diff
@@ -10,99 +10,102 @@
 
 DOCUMENTATION = '''
 ---
-module: cisco_spark
-short_description: Send a message to a Cisco Spark Room or Individual.
+module: cisco_webex
+short_description: Send a msg to a Cisco Webex Teams Room or Individual
 description:
-    - Send a message to a Cisco Spark Room or Individual with options to control the formatting.
+    - Send a msg to a Cisco Webex Teams Room or Individual with options to control the formatting.
 author: Drew Rusell (@drew-russell)
 notes:
   - The C(recipient_id) type must be valid for the supplied C(recipient_id).
-  - Full API documentation can be found at U(https://developer.ciscospark.com/endpoint-messages-post.html).
+  - Full API documentation can be found at U(https://developer.webex.com/docs/api/basics).
 
 options:
 
   recipient_type:
     description:
-       - The request parameter you would like to send the message to.
+       - The request parameter you would like to send the I(msg) to.
```

Comment:
```suggestion
       - The request parameter you would like to send the message to.
```
This is an English word, not the module option.

Reply:
corrected